components/layout/Layout.jsx

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+import React from "react";
+import Header from "./Header";
+import { Footer } from "./Footer";
+
+const Layout = ({ hasFooter, title, children }) => {
+  console.log(hasFooter);
+  return (
+    <>
+      <Header title={title} />
+      <StyledMain hasFooter={hasFooter}>{children} </StyledMain>
```

Comment:
1. `hasFooter` shorthand
2. there is an extra space after children. A single space in this case can actually end up in you DOM. Why it is there would be hard to track down.

```suggestion
      <StyledMain hasFooter>{children}</StyledMain>
```